### PR TITLE
[docs] Guide to creating/editing SMF services in omicron

### PR DESCRIPTION
**Pull request by karencfv**
This is probably incomplete, we can add more stuff down the track. 

Just wanted to add some information while it's fresh in my mind

### CODE REVIEW

File: docs/adding-or-modifying-smf-services.adoc [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    { from = "smf/oximeter", to = "/var/svc/manifest/site/oximeter" },
+    { from = "oximeter/db/schema", to = "/opt/oxide/oximeter/schema" },
+]
+output.type = "zone"
+output.intermediate_only = true
+```
+
+== Sled agent services
+
+To make sure all of the service properties are populated, you'll have to edit the `sled-agent/src/services.rs` file. This is where all of the omicron zones are built.
```
Comment (smklein) [inline docs/adding-or-modifying-smf-services.adoc:69]: Might be worth noting -- these are valuable for properties that "can only be known at run-time"

File: docs/adding-or-modifying-smf-services.adoc [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+= Adding or modifying SMF services
+
+== SMF services
+
+Go to the `smf/` directory, there you will find all current services or you create a new directory if you're working on a new service.
+
+Each directory can contain a myriad of files depending on the service, but only a `manifest.xml` file is necessary. This file contains the complete set of properties associated with the service instance.
+
+== Packaging
```
Comment (smklein) [inline docs/adding-or-modifying-smf-services.adoc:13]: Thanks for documenting these short-comings. These are default decisions that were made ("rust package" comes from "package name", and "service_name" means "zone name" for zones) but this also serves as good data points for "how the manifest could be made more clear + explicit in the future".

**Review by smklein — APPROVED** [review]
Thanks for adding this!

**Comment by karencfv** [comment]
Thanks for the review @smklein :bowing_woman:

**Pull request merged**